index.html

```diff
@@ -42,26 +42,60 @@
+    <a href="#create-thumbnails"><button type="button" class="btn contents-list">Generate image files from a video</button></a>
+    <a href="#animated-gif"><button type="button" class="btn contents-list">Generate an animated GIF</button></a>
     <a href="#create-video"><button type="button" class="btn contents-list">Create a video from image(s) and audio</button></a>
     <a href="#overlay"><button type="button" class="btn contents-list">Overlay timecode or text on a video</button></a>
     <a href="#normalise-audio"><button type="button" class="btn contents-list">Normalize/equalize audio</button></a>
     <a href="#preservation"><button type="button" class="btn contents-list">Preservation tasks</button></a>
     <a href="#test-files"><button type="button" class="btn contents-list">Generate test files</button></a>
     <a href="#repair"><button type="button" class="btn contents-list">Repair a file</button></a>
     <a href="#ocr"><button type="button" class="btn contents-list">Use OCR</button></a>
+    <a href="#concepts"><button type="button" class="btn contents-list">Understand more advanced ffmpeg concepts</button></a>
     <a href="#other"><button type="button" class="btn contents-list">Something else</button></a>
   </div>
 
   <div class="well">
+  <h2 id="getting-started">Getting started with ffmpeg</h2>
+
+  <!-- Basic structure of an ffmpeg command -->
+  <span data-toggle="modal" data-target="#basic-structure"><button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="bottom" title="Basic structure of an ffmpeg command">Basic structure of an ffmpeg command</button></span>
+  <div id="basic-structure" class="modal fade" tabindex="-1" role="dialog">
+    <div class="modal-dialog modal-lg">
+      <div class="modal-content">
+        <div class="well">
+          <h3>Basic structure of an ffmpeg command</h3>
+          <p>At its basis, an ffmpeg command is relatively simple. After you have installed ffmpeg (see instructions <a href="https://avpres.net/FFmpeg/#ch1">here</a>), the program is invoked simply by typing <code>ffmpeg</code> at the command prompt.</p>
+          <p>Subsequently, each instruction that you supply to ffmpeg is actually a pair: a flag, which designates the <i>type</i> of action you want to carry out; and then the specific action itself.</p>
+          <p>For example, with the instruction <code>-i <i>input_file.ext</i></code>, the <code>-i</code> flag tells ffmpeg that you are supplying an input file, and <code>input_file.ext</code> states which file it is.</p>
+          <p><b>Note:</b> flags are always prepended with a hyphen.</p>
+          <p>A very basic ffmpeg command looks like this:</p>
+          <dl>
+            <dt>ffmpeg</dt><dd>starts the command</dd>
+            <dt>-i <i>input_file.ext</i></dt><dd>path and name of the input file<br></dd>
+            <dt><i>-flag some_action</i></dt><dd>tell ffmpeg to do something, by supplying a valid flag and action</dd>
```

Comment:
I did write that exact example, but then took it out - readding!

Reply:
ProRes is a little tricky, because Apple stopped supporting QuickTime on Window, and an archive should use Matroska instead. (More on this in Vienna.)